docs/src/examples/examples_maps.md

```diff
@@ -36,28 +45,33 @@ unemployment = dataset("unemployment.tsv").path
 ```@example
 using VegaLite, VegaDatasets
 
-dataset("zipcodes").path |>
+zipcodes=read(dataset("zipcodes").path,String);
```

Comment:
I think `dataset("zipcodes") |>` should work here, and then we don't need a separate `data` element in the `@vlplot`?

Reply:
Yes, `dataset("zipcodes") |>` works.